vscode/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
I don't think you need to repeat this here if you aren't including it in the package itself.
If you intended to included it, you'll need to add it to the packager data deps

Reply:
(you can also use copy_file like for the image)

Reply:
So I don't think I do need to include this in the package data deps, because it's appearing on the marketplace site when I publish: https://marketplace.visualstudio.com/items/dataform.dataform/license

Reply:
In fact I think what's happening here is that the `vsce publish` command I run takes some info that isn't from the packaged bundle, just from the directory it's in. So I think I need this file here (unless/until we write a bazel publish command)